ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly-4.13-upgrade-from-aro.yaml

```diff
@@ -23,21 +23,7 @@ base_images:
     name: "4.12"
     namespace: ocp
     tag: tools
-  upi-installer:
-    name: "4.11"
-    namespace: ocp
-    tag: upi-installer
 releases:
-  intermediate411:
-    release:
-      architecture: amd64
-      channel: fast
-      version: "4.11"
-  intermediate412:
-    release:
-      architecture: amd64
-      channel: fast
-      version: "4.12"
   latest:
```

Comment:
Though the ARO install does not use the "release:latest" image to run the install, still suggest to update `version` to 4.13 for easy undertstanding.